cpp/frontend_pyreadline.cc

```diff
@@ -0,0 +1,82 @@
+#include "frontend_pyreadline.h"
+
+#include <assert.h>
+
+#include "_build/detected-config.h"
+
+namespace py_readline {
+
+Readline::Readline() : Obj(Tag::Opaque, kZeroMask, sizeof(Readline)) {
```

Comment:
FWIW I started putting `kNoObjLen` for the length instead of `sizeof`

The Cheney allocator needed the length for all objects to copy, while Mark and Sweep doesn't, so it's unused

(the length would only be needed for variable length objects like Slab -- for list and dict)


At some point we should make them all consistent